common/persistence/dataInterfaces.go

```diff
@@ -212,6 +212,11 @@ type (
 		PreviousRangeID int64
 	}
 
+	// CloseShardRequest is used to notify persistence that we're unloading a shard
+	CloseShardRequest struct {
```

Comment:
Personally I wish we didn't have a proto for every single little thing we do.  RequestShardAction with an enum for the action type would be a lot cleaner imo, but I'm pretty sure that ship has sailed :)  Just kvetching.